CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+This release changes the OpenTelemetry Collector `Traces` service endpoint `CHART_NAME-sumologic-otelcol.NAMESPACE`
+to `deprecated`. Service will still work and point to `CHART_NAME-sumologic-otelagent.NAMESPACE`.
```

Comment:
so we deprecate it, but still use it?

Reply:
`otelcol` vs `otelagent`

Reply:
yes, the reason is that there are customers which deployed tracing without `otelagent`s, so to avoid issues with missing data/export errors we routed traffic to `otelagent` daemonset using `otelcol` service.